$OpenBSD: patch-src_calibre_ebooks_pdf_reflow_h,v 1.1 2011/11/26 19:50:08 kili Exp $

With poppler-0.18, the classes Link and AnnotLink had been merged
into AnnotLink. The latter should now contain all functionality of
the former.

--- src/calibre/ebooks/pdf/reflow.h.orig	Fri Sep 23 19:14:20 2011
+++ src/calibre/ebooks/pdf/reflow.h	Thu Nov 17 16:33:49 2011
@@ -244,6 +244,6 @@ class XMLOutputDev : public OutputDev {
     XMLImages *images;
     PDFDoc *doc;
 
-    void process_link(Link* link);
+    void process_link(AnnotLink* link);
 };
 }
